ui/src/main/java/com/epam/eco/kafkamanager/ui/config/DisabledWebSecurityConfig.java

```diff
@@ -30,22 +33,14 @@
 @Configuration
 @EnableWebSecurity
 @Order(1)
-public class DisabledWebSecurityConfig extends WebSecurityConfigurerAdapter {
-
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        HttpSecurityConfigurer.
-            configureLogout(http).
-            logout().
-                logoutSuccessUrl("/"). // just to avoid redirect to default "/login?logout"
-                and().
-            csrf().
-                // https://github.com/thymeleaf/thymeleaf-spring/issues/110
-                // https://github.com/spring-projects/spring-security/issues/3906
-                csrfTokenRepository(new HttpSessionCsrfTokenRepository()).
-                and().
-            authorizeRequests().
-                anyRequest().permitAll();
+public class DisabledWebSecurityConfig  {
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        http.sessionManagement( session->session.sessionCreationPolicy(SessionCreationPolicy.ALWAYS))
```

Comment:
As I understand, this settings will force spring to create session each time when an user hit HTTP-request. Can it, somehow, influence heap size or security concerns?

Reply:
It is necessary for using thymileaf. Because it has controversy between session creation from SpringSecurity and thymileaf. Spring security try to add csrf token to every session, except sessions created by thymileaf. And if cause exception: Cannot create a session after the response has been committed.

Link:
https://stackoverflow.com/questions/52982246/spring-thymeleaf-throws-cannot-create-a-session-after-the-response-has-been-com